src/NotepadNext/NotepadNextApplication.cpp

```diff
@@ -387,7 +387,7 @@ QString NotepadNextApplication::detectLanguageFromContents(ScintillaNext *editor
 
 void NotepadNextApplication::loadSystemDefaultTranslation()
 {
-    loadTranslation(QLocale::system());
+    loadTranslation(QLocale(QLocale::system().name()));
```

Comment:
Only thing I can suggest is add a very brief one-liner comment above this since it is not immediately obvious why the `QLocale::system()` cannot be used.

Reply:
Good suggestion! Comment added.